src/Pages/Utils/FormatDate.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+
+class FormatDate extends PureComponent {
+  static props = {
+    date: PropTypes.instanceOf(Date).isRequired
+  };
+
+  format = (date) => {
+    return [
+      this.getDayName(date.getDay()),
+      ' ',
+      date.getDate(),
+      ' de ',
+      this.getMonthName(date.getMonth()),
+      ' de ',
+      date.getFullYear()
+    ].join(' ');
+  };
+
+  getDayName = (day) => {
```

Comment:
This should be expressed as an array of weekdayNames and have a fn that receives weekDayName and return an `weekDayNames.indexOf(weekDayName)` (consider moving this to an external module)